glide.lock

```diff
@@ -1,5 +1,5 @@
-hash: bd44847c5d8eb9a431262aa7dc304c235ed5fd59a5e106b827a4c171e500c670
-updated: 2017-07-20T09:42:45.593735247+08:00
+hash: 8a6d0301e099f3d4860e9a4ae741b65c82e318b4dc969652589df0754f96b6ae
+updated: 2017-06-29T15:09:15.857129041+08:00
```

Comment:
My PR came up earlier, which led to this situation when `rebase` was done. Now updated.